src/main/resources/data.sql

```diff
@@ -1,7 +1,18 @@
 -- TODO: 기능 구현에 필요한 내용을 추가하거나 수정하세요.
-CREATE TABLE PLAY_RESULT (
-    id          INT         NOT NULL AUTO_INCREMENT,
+DROP TABLE racing_game if exists;
+DROP TABLE racing_car if exists;
+
+CREATE TABLE racing_game (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS racing_game (
```

https://dev.mysql.com/doc/refman/8.0/en/create-table.html
IF NOT EXISTS로 합쳐볼 수 있겠네요!